_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_ufo_crash.dmm

```diff
@@ -6,7 +6,9 @@
 /turf/simulated/floor/plating/asteroid/basalt/lava_land_surface,
 /area/lavaland/surface/outdoors)
 "c" = (
-/obj/machinery/abductor/experiment,
+/obj/machinery/abductor/experiment{
+	stat = 1
+},
```

Comment:
make this a subtype instead, `abductor/experiment/broken` or something

Reply:
True, a second...